src/rust/zh.dmzj/Cargo.toml

```
@@ -0,0 +1,28 @@
+[package]
+name = "dmzj-aidoku"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib"]
+
+[profile.dev]
+panic = "abort"
+
+[profile.release]
+panic = "abort"
+opt-level = "s"
+strip = true
+lto = true
+
+[dependencies]
+aidoku = { git = "https://github.com/Aidoku/aidoku-rs/" }
+base64ct = "*"
+rsa = {git = "https://github.com/RustCrypto/RSA", default-features = false}
```

```suggestion
rsa = { version = "0.6.1", default-features = false }
```
Why not use the `crates.io` crate?